azure/packages/azure-client/src/interfaces.ts

```diff
@@ -31,6 +31,11 @@ export interface AzureClientProps {
 	readonly configProvider?: IConfigProviderBase;
 
 	readonly summaryCompression?: boolean | ICompressionStorageConfig;
+
+	/**
+	 * Optional. True if the container is ephemeral
```

Comment:
Can we be more explicit about what we mean by "ephemeral" here? I don't think this is necessarily obvious. Examples (using `@example` blocks) would also be useful.

Reply:
I would also recommend adding `@defaultValue false` to this.